### DISCUSSION TITLE
Auto-import when running in interactive notebook

### DISCUSSION THREAD

**Discussion by maximilianmordig** [microsoft/pylance-release#6171] [Enhancement]
Running in an interactive notebook with `#%%` cells, I frequently add auto-imports (via Cmd+.), so it adds them to the top of the file. If the notebook is long, I have to scroll up, execute the cell, then find the cell I was working at again.
It would be nice if it could automatically import the module, e.g. by evaluating the import in a separate cell.

**Comment by StellaHuang95** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
